rls-quick-start-guide.md

```diff
@@ -0,0 +1,51 @@
+# RLS Quick-Start Guide
+
+Hello, RLS members new and old! Welcome to the Rands Leadership Slack quick-start guide 🙂 
+
+RLS is a big community! The last time Rands made an #announcement, RLS membership was the same size as East Peoria, Illinois with 22,620 people. As RLS continues to grow, it can be harder to figure out where to get started. This quick-start guide is intended to give you that starting point to exploring and participating in this community. 
+
+And please be assured that we do want you - _yes, you!_ - to participate. You are not too junior, or too new, or not leader-y enough to contribute. Our community benefits from being able to discuss and learn from all of our perspectives and experiences, and it's what we love about being in RLS.
+
+## Code of Conduct
+
+Please make sure you read and understand the RLS [Code of Conduct](https://github.com/randsleadershipslack/documents-and-resources/blob/main/code-of-conduct.md).
+
+## Channels of Interest
+
+Rands Leadership Slack has many (many) channels. We could not hope to describe all of them here, but this should help give you a starting point for finding the channel you may be looking for.
+
+- **`#how-to-rands`:** Not sure which channel is the best place for your question or post? Ask for directions here. 
+- **`#rands-*`:** Channels with the `rands-` prefix are related to the operating and particpating in RLS, such as `#rands-admins`, `#rands-slack-culture`, and `#rands-slack-rules`.
+- **`#help-and-advice`:** If there's not a more specific channel that's relevant to your topic, you can always ask in this channel. 
+- **`#anonymous-help-and-advice`:** For various reasons, there are times where someone may not feel comfortable asking for advice publicly in RLS. This channel was created to support those times and allow members to create an anonymous top-level post using a channel workflow. The donwside is that it can remove the ability to have fuller discussions, since the original poster cannot respond anonymously in threads to participate.
```

Comment:
typo: `s/donwside/downside/g`
Subjective but I find "fuller" to be odd, maybe "deeper"?